docs/demos/Cluster.story.tsx

```diff
@@ -467,18 +467,40 @@ export const CustomLabel = () => (
     draggable
     edges={clusterEdges}
     clusterAttribute="type"
-    renderClusterLabel={({ label, opacity, fontUrl }) => (
-      <Billboard position={[0, 0, 1]}>
-        <Svg src={demonSvg} />
-        <Text
-          font={fontUrl}
-          fontSize={12}
-          color={new Color('#2A6475')}
-          fillOpacity={opacity}
-        >
-          Custom {label}
-        </Text>
-      </Billboard>
+    onRenderCluster={({
+      label,
+      opacity,
+      offset,
+      rad,
+      padding,
+    }) => (
+      <>
+        <mesh>
+          <ringGeometry attach="geometry" args={[offset, rad + padding, 128]} />
+          <a.meshBasicMaterial
+            attach="material"
+            color="gray"
+            transparent={true}
+            depthTest={false}
+            opacity={opacity}
+            side={DoubleSide}
+            fog={true}
+          />
+        </mesh>
+        <a.group position={label.position}>
```

Comment:
Couldn't this use `Label` directly?

Reply:
I've added SVG icon for label, that why it not just Label component. Maybe need to extend label component to define start/end icon (optionally)